src/style.css

```diff
@@ -0,0 +1,68 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir.ttc");
+  font-style: normal;
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.big-searchbar,
+.small-searchbar {
+  margin-top: 20px;
+}
+
+.search__input {
+  font-family: "Avenir-Heavy", Helvetica, sans-serif;
+  font-size: 16px;
+  width: 100%;
+  height: 70px;
+  background: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-position: 26px 25px;
+  text-indent: 62px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.smallSearch__input {
```

Comment:
PAy attention to my comments, I've commented about consistent namings and you've forgot to change this one